Lib/test/test_logging.py

```diff
@@ -4487,6 +4496,17 @@ def test_multiprocessing(self):
             if multiprocessing_imported:
                 import multiprocessing
 
+    def test_multiprocessing_again(self):
```

Comment:
I prefer to remove this change, it's unrelated to the logging module.

Reply:
I will make one more version to isolate the whole thing into `test_multiprocessing`. Let's see if that is going to be better. 👍